### PR TITLE
Preventing "onbeforeunload" on "mailto" links

### PR DESCRIPTION
**Pull request by bennyn**
## Type of change

Preventing "onbeforeunload" on "mailto" links.

### CODE REVIEW

**Comment by lipis** [comment]
Interesting :)

File: app/page/index.html [hunk 1]
```diff
@@ -21,5 +21,6 @@
     #include('partials/template-user-profile.htm')
     #include('partials/template-confirm.htm')
     #include('partials/template-message.htm')
+    <iframe name="prevent-onbeforeunload" style="visibility:hidden; position:absolute;"></iframe>
```
Comment (gregor) [inline app/page/index.html]: Can we put those in our style files?

File: app/page/template/partials/template-user-profile.htm [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <div class="user-profile-default">
       <div class="user-profile-header">
         <div class="name popover-title" data-bind="text: user().name(), attr: {'data-uie-uid': user().id, 'data-uie-value': user().name()}" data-uie-name="status-user"></div>
-        <a class="email popover-meta" tabindex=-1 data-bind="text: user().email(), attr: {'href': 'mailto:' + user().email(),'data-uie-uid': user().id, 'data-uie-value': user().email()}" data-uie-name="status-user-email"></a>
+        <a class="email popover-meta" tabindex=-1 data-bind="text: user().email(), attr: {'href': 'mailto:' + user().email(),'data-uie-uid': user().id, 'data-uie-value': user().email(), 'target': 'prevent-onbeforeunload'}" data-uie-name="status-user-email"></a>
```
Comment (gregor) [inline app/page/template/partials/template-user-profile.htm]: No need to do this in the data-bind

Comment (bennyn) [inline app/page/template/partials/template-user-profile.htm]: Where would you put it?

File: app/page/index.html [hunk 3]
```diff
@@ -21,5 +21,7 @@
     #include('partials/template-user-profile.htm')
     #include('partials/template-confirm.htm')
     #include('partials/template-message.htm')
+
+    #include('prevent-onbeforeunload.htm')
   </body>
```
Comment (herrmannplatz) [inline app/page/index.html:26]: 👌

**Pull request merged**